delivery/src/on_package_created/main.py

```diff
@@ -22,6 +24,7 @@
 table = dynamodb.Table(TABLE_NAME) # pylint: disable=invalid-name,no-member
 logger = Logger() # pylint: disable=invalid-name
 tracer = Tracer() # pylint: disable=invalid-name
+metrics = Metrics(namespace="ecommerce.deliveries", service="delivery")
```

Comment:
Tiny nitpick, but since the service name is "ecommerce.delivery", having a name of "deliveries" here would create inconsistencies with the rest of the service.

Reply:
Thanks for pointing that out, I've made the update.  I used "deliveries" to be consistent with "ecommerce.orders" for the Order service